cwms-data-api/src/main/java/cwms/cda/data/dao/ClobDao.java

```diff
@@ -134,10 +134,10 @@ public Clobs getClobs(String cursor, int pageSize, String officeLike,
                         includeValues ? v_clob.VALUE : DSL.inline("").as(v_clob.VALUE)
                 )
                 .from(v_clob)
-                //.innerJoin(forLimit).on(forLimit.field(v_clob.ID).eq(v_clob.ID))
                 .join(v_office).on(v_clob.OFFICE_CODE.eq(v_office.OFFICE_CODE))
                 .where(JooqDao.caseInsensitiveLikeRegex(v_clob.ID,idRegex))
                 .and(DSL.upper(v_clob.ID).greaterThan(clobCursor))
+                .and(officeLike == null ? noCondition() : DSL.upper(v_office.OFFICE_ID).like(officeLike.toUpperCase()))
```

Comment:
This should be caseInsensitiveRegexLike -right?
Paging should also order by office and clobId.
Probably need test
Is there an approved issue for these changes?

Reply:
added tests, made office id support regex, ordered the page by office id and clob id.